src/content/lodash/object/pick/lodash.js

```diff
@@ -0,0 +1,10 @@
+import { pick } from 'lodash'
+
+const object = {
+  a: 1,
+  b: 2,
+  c: 3,
+}
+
+module.exports = pick(object)
```

Comment:
I believe this should be `pick(object, ['a', 'c']);`

other than that, it looks good to me

Reply:
oh yes of course. thanks!